spec/components/permissions_list_component_spec.rb

```diff
@@ -13,17 +13,36 @@
 
   it 'renders permissions' do
     permission_model = create(:provider_permissions, manage_organisations: true)
-    result = render_inline(described_class.new(permission_model))
+    result = render_inline(described_class.new(permission_model, user_is_viewing_their_own_permissions: false))
 
     expect(result.css('li').text).to include('Manage organisations')
-    expect(result.css('li').text).not_to include('View applications only')
+    expect(result.css('li').text).not_to include('The user can only view applications')
+  end
+
+  describe 'rendering View applications only' do
+    it 'shows an appropriate message when the user is viewing another user’s permissions' do
+      permission_model = create(:provider_permissions)
+      result = render_inline(described_class.new(permission_model, user_is_viewing_their_own_permissions: false))
+
+      expect(result.css('li').text).to include('The user can only view applications')
+      expect(result.css('li').text).not_to include('Manage organisations')
+      expect(result.css('li').text).not_to include('Make manage users')
+      expect(result.css('li').text).not_to include('Make decistions')
+      expect(result.css('li').text).not_to include('View safeguarding information')
```

Comment:
Super minor, shouldn't this be 'Access safeguarding information'?

Reply:
Yes! Will amend